k8s/scalyr-agent-2-configmap.yaml

```diff
@@ -15,4 +15,4 @@ data:
   #
 metadata:
   name: scalyr-config
-  namespace: default
+  namespace: scalyr
```

Comment:
Argh, won't this break the current instructions?  It should remain in the default namespace.

Reply:
I think need to comment this out with an instruction above it to uncomment it to create the config map in the scalyr namespace.  Then in your new instructions, make sure you tell them to do that.

Reply:
Well the current instructions don't even reference the file here, they tell you to create the config file yourself and give an example right in the docs, so I don't think this should affect anything.

Reply:
Ah, good catch.  Thanks for looking at that.  Yep, never mind then.